dockers/docker-sonic-mgmt-framework/Dockerfile.j2

```diff
@@ -15,6 +15,8 @@ RUN pip2 install --upgrade 'pip<21'
 RUN apt-get purge -y python-pip
 RUN pip2 install setuptools==40.8.0
 RUN pip2 install wheel==0.35.1
+RUN pip2 install PyYAML==5.4.1 --no-build-isolation
```

Comment:
>PyYAML [](http://example.com/codeflow?start=17&length=6)

I see it is installed in docker-config-engine-buster, and this Docker is FROM docker-config-engine-buster. Why install again?

Reply:
In docker-config-engine-buster only for python3, no python2.